.github/workflows/app.yml

```diff
@@ -54,6 +58,10 @@ jobs:
         uses: actions/checkout@v3
         with:
           submodules: true
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
```

Comment:
Needing to add this isn't a great side effect, why not add an environment variable to skip forge install rather? (or use the CI environment variable)

Reply:
@JasoonS I like that idea – made this a draft for now, I will refactor the `postinstall` to not run `forge install` in CI.

Reply:
@JasoonS thanks again for the feedback, updated to reflect it!

Reply:
Given that this repo expects foundry (even if not used in the isolated context of the app package), I think this kind of change is fine/makes sense, and reflects what your local developer environment might have anyway.

We might consider adding this in as part of #36